src/sage/interfaces/singular.py

```diff
@@ -1206,19 +1206,22 @@ def current_ring(self):
 
         EXAMPLES::
 
+            sage: import re
             sage: r = PolynomialRing(GF(127),3,'xyz', order='invlex')
-            sage: r._singular_()
+            sage: rsing = r._singular_()
+            sage: print(re.sub('ordering rp', 'ordering ip', rsing._repr_()))
             polynomial ring, over a field, global ordering
             //   coefficients: ZZ/127
             //   number of vars : 3
-            //        block   1 : ordering rp
+            //        block   1 : ordering ip
```

Comment:
```suggestion
            //        block   1 : ordering ...
```
wouldn't this make it simpler?

Reply:
I considered this, but do we really want to skip testing that the correct ordering is used here?

Reply:
We really need a mechanism to support backward compatibility. Sprinkling examples with regular expressions to fix stuff like this is quite ugly in terms of documentation.

Similar problems we have right now with gap 4.13 (#37616, see also https://github.com/sagemath/sage/pull/37590#issuecomment-2013350669) and all over the place with results that have to be sorted to compare, etc.

For numerical noise there's the `# tol` tag which affects how the output of a doctest is compared. For order, I think a tag `# random_order` may be preferable to having to sort in the example. What could we do for this case?

An alternative is to add code in `SingularElement._repr_()` so the output of old singular is changed into the output of new singular (i.e. replace `ordering rp` with `ordering ip`). But this seems a bit hacky...